CairoMakie/src/CairoMakie.jl

```diff
@@ -2,9 +2,9 @@ module CairoMakie
 
 using Makie, LinearAlgebra
 using Colors, GeometryBasics, FileIO
-import SHA
```

Comment:
I'm not a big fan of this :D Can we specify that in the formatter.toml?

Reply:
That is the consequence of https://github.com/MakieOrg/Makie.jl/blob/f124a016d2662ec12d31dac4098b7a92622bd3b1/.JuliaFormatter.toml#L3.

Remove ?

Reply:
Haha, surely! Totally forgot about the original content of that file!

Reply:
What I would propose is try to use the minimal number of options with the "yas" style, and add them after if needed :)